### PR TITLE
Add PATH outputs

### PR DESCRIPTION
**Pull request by profxj**
As titled, this PR includes the PATH estimates for FRB hosts.

Need to add tests and a bit of docs before merging.

### CODE REVIEW

**Comment by profxj** [comment]
Tests now passing:

========== 59 passed, 1 skipped, 2224 warnings in 1157.66s (0:19:17) ==========

File: docs/galaxies.rst [hunk 1]
```diff
@@ -123,4 +123,24 @@ you Instantiate the class by the FRB name::
...
 
 Of course, a previously generated JSON file must already
-have been archived.
\ No newline at end of file
+have been archived.
+
+PATH
+----
+
+A subset of the FRB Host galaxies have been analyzed using
```
Comment (tarraneh) [inline docs/galaxies.rst:131]: using >> using the

File: frb/builds/build_path.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    """Main method for generating a Host JSON file
+
+    Args:
+        frb_list (list): List of FRB names from the database
+        host_coords (list): List of host galaxy coords fom the database
+        prior (dict):
+            Prior for PATH
+        override (bool, optional): Attempt to over-ride errors. 
+            Mainly for time-outs of public data. Defaults to False.
+        tol (float, optional):  Tolearance for a match to the expected host
```
Comment (tarraneh) [inline frb/builds/build_path.py:44]: Tolearance >> Tolerance

File: frb/builds/build_path.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    return df
+
+def main(options:str=None, override:bool=False):
+    """ Driver of the analysis
+
+    Args:
+        options (str, optional): [description]. Defaults to None.
+        override (bool, optional): [description]. Defaults to False.
+    """
+    # Parse optionsd
```
Comment (tarraneh) [inline frb/builds/build_path.py:99]: optionsd >> options

File: frb/galaxies/utils.py [hunk 4]
```diff
@@ -207,3 +234,18 @@ def load_f_mL():
...
 
+
+def load_PATH(PATH_root_file:str='adopted.csv'):
+    """Load up the PATH table
+
+    Args:
+        PATH_root_file (str, optional): [description]. Defaults to 'adopted.csv'.
+
+    Returns:
+        pandas.DataFrame: Table of galaxy coordiantes and PATH results
```
Comment (tarraneh) [inline frb/galaxies/utils.py:245]: coordinates

**Review by tarraneh — CHANGES_REQUESTED** [review]
Minor changes

File: frb/associate/frbassociate.py [hunk 5]
```diff
@@ -260,7 +142,7 @@ def cut_candidates(self, plate_scale, bright_cut=None, separation=None):
 
         """
 
-        # Zero point
+        # Plate scale
```
[the remark this answers is not in the document]
Comment (profxj) [reply frb/associate/frbassociate.py:145]: done

File: frb/galaxies/utils.py (hunk 4 above)
Comment (profxj) [reply frb/galaxies/utils.py:245]: fixed

**Comment by profxj** [comment]
============ 57 passed, 3 skipped, 31 warnings in 118.22s (0:01:58) ============

File: frb/builds/build_path.py (hunk 2 above)
Comment (profxj) [reply frb/builds/build_path.py:44]: fixed

**Review by tarraneh — APPROVED** [review]
All good

**Comment by profxj** [comment]
Merging!

**Pull request merged**